src/lib.rs

```diff
@@ -243,6 +251,8 @@ impl std::error::Error for Error {
         match self {
             Backend(error) => Some(error),
             Prctl(error) => Some(error),
+            Seccomp(error) => Some(error),
+            ThreadSync(_) => None,
```

Comment:
I'm not sure if there's something better I can do here.
`ThreadSync(pid) -> Some(pid)` doesn't work because i64 doesn't impl Error.